client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementEntry.vue

```diff
@@ -35,6 +36,7 @@ All rights reserved
         <dp-datepicker
           id="authoredDateDatepicker"
           class="o-form__control-wrapper"
+          data-cy="statementEntry:authoredDate"
```

Comment:
I'm not sure about this but I remember the `data-cy` attributes should best be unique, however this shall not block this PR hence can be viewed as nit.

Reply:
Yeah, but these are in two if - else blocks. I figured this would be ok, but I never worked with cypress. Do you know if it makes problems?

Reply:
Ah I have overseen this in the diff sorry, my bad. My guess is this is not problematic at all since in the browser there is only one of both components so each attribute should be unique when cypress performing tests in a simulated environment.